### ISSUE TITLE
FFB methods add extra axes

### ISSUE THREAD

**Issue by janden** [ComputationalCryoEM/ASPIRE-Python#114]
For example,
```
sz = (8, 8, 8)
basis = FFBBasis3D(sz)
v = np.random.randn(basis.count)
assert basis.evaluate(v) == sz + (1,)
```
and
```
x = np.random.randn(*basis.sz)
assert basis.evaluate_t(x) == (basis.count, 1)
```
This gives rather confusing behavior when trying to run `expand` and other functions.

**Comment by junchaoxia** [comment]
I think the problem has bee fixed in an old commit 44d685d.  I am able to go though the code below without problem. 

> sz = (8, 8, 8)
> basis = FFBBasis3D(sz)
> v = np.random.randn(basis.count)
> assert basis.evaluate(v).shape == sz  

and 

> x = np.random.randn(*basis.sz)
> assert basis.evaluate_t(x).shape == (basis.count)

Closing Now.

**Comment by janden** [comment]
That commit is older than the issue, so it couldn't have fixed the bug. More likely, it's moving the `expand` function out of `FFBBasis3D` in 64b6ee3be5666f8f18354357584b02aa27d8f29b that did it.
